Example/URBNSwiftyConvenience/ViewController.swift

```diff
@@ -12,10 +12,13 @@ import URBNSwiftyConvenience
 class ViewController: UIViewController {
     
     var animateBorder: UIView?
+    private let childViewController = ChildViewController()
```

Comment:
Why the instance variable

Reply:
The instance variable is used in:
```
@objc private func addChildAction() {
        embedChildViewController(childViewController)
  }
```

Reply:
yes, but why?

Reply:
This is so I can add the view from the childVC to the parentVC via the button? Is this not ideal?

Reply:
you can do it all within the `addChildAction()` button